docs/guide/checkout/local_orders.md

```diff
@@ -0,0 +1,75 @@
+# Local orders
+
+If the shop is not connected to an ERP system, the customer can still follow the order process
+and create a local order that generates an invoice PDF and sends it by mail.
+
+The order is stored locally in the shop.
+
+## Configuring local orders
+
+By default the shop sends the order after the checkout process to ERP.
+But you can also disable using ERP and use the local order process instead.
+
+To do this, disable sending the order to ERP:
+
+``` yaml
+parameters:
+    siso_local_order_management.default.send_order_to_erp: false
+```
+
+To avoid sending the order to ERP, the `LocalOrderRequiredException` exception is thrown that interrupts the default process (sending to ERP).
+The message and the order are logged as failed.
+
+The shop then reacts like an ERP system, confirms the order and returns an order ID.
+
+The response has the same structure as a response returned from ERP, so no additional changes in the template are required.
```

Comment:
```suggestion
The response has the same structure as the response returned from ERP, so no additional changes in the template are required.
```
As this is a specific response (referred to by "returns an order ID" in the previous line)